src/test/kotlin/com/expediagroup/sdk/core/config/ConfigurationDefinitionTest.kt

```diff
@@ -44,10 +46,10 @@ class ConfigurationDefinitionTest {
 
         assertDoesNotThrow {
             val configurationKey = definition.get(configurationKeyName)
-            assertEquals(configurationKey.name, configurationKeyName)
-            assertEquals(configurationKey.documentation, configurationKeyDocumentation)
-            assertEquals(configurationKey.type, configurationKeyType)
-            assertEquals(configurationKey.importance, configurationKeyImportance)
+            assertEquals(configurationKeyName, configurationKey.name)
```

Comment:
Thanks for doing these changes, but again, please make sure to do such changes in seperate PRs.

Reply:
That is noted. Thanks for alerting me